ChangeLog.md

```diff
@@ -4,7 +4,8 @@
 
 ### Performance enhancements
 
-* Updated to Python 3.8
+* Updated to Python 3.8.
+* Update MAFFT to >=7.39.
```

Comment:
@antgonza just to confirm, the code actually specifies >=7.394. Not sure if MAFFT has a 7.393, 7.392, ...